### DISCUSSION TITLE
0.26.4, on single pages, the content is now aligned to the left instead of being centered

### DISCUSSION THREAD

**Discussion by vitabef** [gethinode/hinode#1126] [Help] [answered]
In version 0.26.4, on single pages, the content is now aligned to the left instead of being centered. This is likely due to the missing sidebar?

**Comment by markdumay** [comment, marked as answer]
That's correct, you can [enable/disable the sidebar](https://gethinode.com/docs/configuration/navigation/) globally by setting the following value in `params.toml`:

```toml
[navigation]
    sidebar = true
```

**Reply by vitabef** [reply]
Oh, I see, sorry, I didn’t realize that right away ;)
